core/arch/riscv64/riscv64.asm

```diff
@@ -264,23 +264,34 @@ GLOBAL_LABEL(_dynamorio_runtime_resolve:)
  */
         DECLARE_FUNC(dynamorio_clone)
 GLOBAL_LABEL(dynamorio_clone:)
-/* FIXME i#3544: Not implemented */
+        addi     ARG2, ARG2, -16 /* Description: newsp = newsp + 16. */
+        sd       ARG1, 8 (ARG2)
+        sd       ARG6, 0 (ARG2) /* The func is now on TOS of newsp. */
+        li       SYSNUM_REG, SYS_clone /* All args are already in syscall registers.*/
+        ecall
+        bnez     ARG1, dynamorio_clone_parent
+        ld       ARG1, 0 (sp)
+        ld       ARG2, 8 (sp)
+        addi     sp, sp, 16
+        jalr     ARG1
+        jal      GLOBAL_REF(unexpected_return)
+dynamorio_clone_parent:
         ret
         END_FUNC(dynamorio_clone)
 
         DECLARE_FUNC(dynamorio_sigreturn)
 GLOBAL_LABEL(dynamorio_sigreturn:)
-        li        SYSNUM_REG, SYS_rt_sigreturn
+        li       SYSNUM_REG, SYS_rt_sigreturn
         ecall
-        jal       GLOBAL_REF(unexpected_return)
+        jal      GLOBAL_REF(unexpected_return)
         END_FUNC(dynamorio_sigreturn)
 
         DECLARE_FUNC(dynamorio_sys_exit)
 GLOBAL_LABEL(dynamorio_sys_exit:)
-        li        a0, 0 /* exit code */
-        li        SYSNUM_REG, SYS_exit /* SYS_exit number */
+        li       a0, 0 /* exit code */
+        li       SYSNUM_REG, SYS_exit /* SYS_exit number */
```

Comment:
Remove the redundant comment `/* SYS_exit number */`

Reply:
Done